sql/src/test/java/org/apache/druid/sql/calcite/QueryVerification.java

```diff
@@ -47,7 +47,12 @@ public QueryTestRunner.QueryVerifyStep make(QueryTestRunner.BaseExecuteQuery exe
     {
       return () -> {
         for (QueryTestRunner.QueryResults queryResults : execStep.results()) {
-          verifier.verifyResults(queryResults);
+          try {
+            verifier.verifyResults(queryResults);
+          }
+          catch (Exception e) {
+            throw new RuntimeException("Exception during verification!", e);
```

Comment:
Can use a druid exception here

Reply:
I was looking around what else to use - but since this is part of the test framework and `DruidException` wants "Persona" and "Category" which is clearly unrelated as an `Exception` here means that the test most likely have crashed...
```
            throw DruidException.forPersona(Persona.DEVELOPER)
                .ofCategory(Category.UNCATEGORIZED)
                .build(e, "Exception during verification!");
```
I could probably use  `Assert.fail`  here - but the junit4 `Assert.fail` does not accept an `Exception` so it would not retain the stacktrace... junit5 has a method like that - but we don't yet have that here (...and since the test most likely crashed; and not failed using that would kinda miss its usecase)

Letting out the `Exception` at this point kinda leads to a shotgun surgery  (although I do thing that every test method should be allowed to just let checked `Exception`-s out)

I was looking around what other exceptions are out there...but didn't found a better match than retaining to wrap the checked `Exception` into a plain `RuntimeException` ; to let it out.